changelog.d/20231009_103807_maria_add_aws_s3_prefix_support.md

```diff
@@ -0,0 +1,4 @@
+Added
```

Comment:
```suggestion
### Added
```

Also, just FYI, you can have multiple entries in the same fragment file (multiple categories, even).